app/views/reports/itt_new_starter_data_signoff.html.erb

```diff
@@ -0,0 +1,41 @@
+  <div class = govuk-grid-column-two-thirds-from-desktop>
+  <h1 class="govuk-heading-l">Export new trainee data for sign off for the 2022 to 2023 academic year</h1>
+    <p class="govuk-body">This export will show the new trainee data you’ve provided to the DfE for trainees who’ve started their initial teacher training (ITT) in the 2022 to 2023 academic year, and have a trainee start date on or before the second Wednesday of October 2022 (sometimes called the census date).</p>
+    <div class="govuk-inset-text">For the 2022 to 2023 academic year, the ITT census date is Wednesday 12 October 2022.</div>
+    <p class="govuk-heading-m">What you must do to sign off your new trainee data</p>
+    <p class="govuk-body">Use this export to check you’ve provided accurate data for all your new trainees. You should check for any errors or missing trainees.</p>
+    <p class="govuk-body">Once you’ve checked your data, a senior person from your organisation (preferably a different person to who submitted the data) should sign it off <a href="#" class="govuk-link">using this form</a>.</p>
+    <p class="govuk-body">You must sign off your data on or before 31 October 2022. Not submitting and signing off new trainee data by this date will mean that we (the DfE) do not have accurate data on your trainees and they may be excluded from the ITT census publication.</p>
+    <h2 class="govuk-heading-m">How new trainee data relates to the ITT census publication</h2>
+    <p class="govuk-body">After signing off your new trainee data, it gets analysed and filtered for the ITT census publication. </p>
+    <p class="govuk-body">Not all your new trainees in this export will be included in the ITT census publication. To understand what data will be used, read the <a href="https://explore-education-statistics.service.gov.uk/methodology/initial-teacher-training-census-methodology" class="govuk-link">Initial Teacher Training Census methodology</a>.</p>
+    <h2 class="govuk-heading-m">New trainees without a start date</h2>
+    <p class="govuk-body">Any new trainee who has a course start date on or before 12 October 2022, but does not have a trainee start date, will be included in this export.</p>
+    <p class="govuk-body">If a new trainee starts their course on or before 12 October 2022, you should add their trainee start date as soon as possible. Not adding this means we will not know if the trainee has started their course, which could affect funding payments (if the trainee is eligible for funding).</p>
+    <h2 class="govuk-heading-m">About this export</h2>
+    <p class="govuk-body">This export includes all trainees:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        with their academic start year in 2022 to 2023
+      </li>
+      <li>
+        with an ITT course start date on or before the ITT census date
+      </li>
+      <li>
+        with a trainee start date on or before the ITT census date
+      </li>
+    </ul>
```

Comment:
I think all these dates should by dynamic for the page we're looking at and the academic year.